crates/lox_core/src/ephemeris/daf_spk/api.rs

```diff
@@ -0,0 +1,125 @@
+use super::parser::{DafSpkError, Spk, SpkSegment, SpkType2Array, SpkType2Record};
+
+type Position = (f64, f64, f64);
+type Epoch = f64;
+type Body = i32;
+
+impl Spk {
+    fn find_segment(&self, origin: Body, target: Body) -> Result<(&SpkSegment, i32), DafSpkError> {
+        let mut sign = 1;
```

Comment:
I'd consider using an `isize` to represent the sign, simply because the system integer type tends to be the idiomatic return type for sign values, lengths, etc. Since the `&SpkSegment` reference will also have the system word size, the overall size of the return value won't change, even on 64-bit systems (the remaining 32 bits would just get padded out).

Reply:
Used isize.